common-custom-user-data-scripts/gradle-common-custom-user-data.gradle

```diff
@@ -291,7 +291,7 @@ boolean isTravis() {
     System.getenv('TRAVIS_JOB_ID')
 }
 
-boolean isGitInstalled() {
+static boolean isGitInstalled() {
```

Comment:
Please make the 8 isXYZ methods above static too

Reply:
Done.